solutions/standard/main.tf

```diff
@@ -0,0 +1,32 @@
+#######################################################################################################################
+# Local block
+#######################################################################################################################
+
+locals {
+  prefix = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
+}
+
+#######################################################################################################################
+# Resource Group
+#######################################################################################################################
+
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  resource_group_name          = !var.use_existing_resource_group ? (var.prefix != null ? "${var.prefix}-${var.resource_group_name}" : var.resource_group_name) : null
```

Comment:
You are not utilizing the `local.prefix`. 
Can you update this to below? 
`resource_group_name          = var.use_existing_resource_group ? null : try("${local.prefix}-${var.resource_group_name}", var.resource_group_name)`